lib/test/test_hamburger.rb

```diff
@@ -184,6 +184,6 @@ def test_header_content_nobody_en
 
   def test_header_content_nobody_nonen
     contents = Hamburger.get_header_contents({user_type: nil, language: "fr"})
-    assert_includes_id contents, "header-projects"
+    assert_includes_id contents, "header-schools"
```

Comment:
Just checking if this makes sense in non-en? /administrators seems pretty US-centric?

Reply:
I was actually think today about how we should add a callout on that page to the international partner pages. 

However we should not special case the header navigation for that. I don't want to go back down that path now that they are unified.

Reply:
I added a ticket for Moshe to look at it